modules/p2/ding_serendipity/plugins/content_types/serendipity_ting_object.inc

```diff
@@ -32,7 +32,7 @@ function ding_serendipity_serendipity_ting_object_admin_info($subtype, $conf, $c
 function ding_serendipity_serendipity_ting_object_content_type_render($subtype, $conf, $panel_args, $context) {
   // Only show on front if serendipity is enabled for the frontpage.
   // See ding_frontpage config at /admin/config/ding/frontend.
-  if (drupal_is_front_page() && !variable_get('ding_frontpage_serendipity') && variable_get('ding_frontpage_promoter') != 'serendipity') {
+  if (isset($conf['selected_keys']['frontpage']) && !variable_get('ding_frontpage_serendipity') && variable_get('ding_frontpage_promoter') != 'serendipity') {
```

Comment:
@BirkAndMe has explained that this is caused by panes loaded through lazy_pane are not aware of where they are being inserted. Thus we test for a setting instead.